crates/typed-store/src/rocks/mod.rs

```diff
@@ -236,6 +244,55 @@ impl RocksDB {
     }
 }
 
+pub enum RocksBatch {
```

Comment:
small nit: could we rename it to `RocksDBBatch` to be consistent with rest of the naming in this crate